### DISCUSSION TITLE
text is blury / cut off

### DISCUSSION THREAD

**Discussion by B-Mqn** [CorsixTH/CorsixTH#2360] [General Diagnostics]
### Describe the issue

text in everything is hard to read or actually unreadable

https://drive.google.com/file/d/13KB7rhmGvy6yiJm5qQ9ap6mX-SePRqVz/view?usp=sharing

### Steps to Reproduce

open game.
 its in all menus at all resolutions 

### Expected Behaviour

to be able to read the text

### Save Game

no

### CorsixTH Version

0.66 - CorsixTHInstaller.exe 

### Operating System

windows 10

### Theme Hospital Version

CD

**Comment by lewri** [comment]
> https://drive.google.com/file/d/13KB7rhmGvy6yiJm5qQ9ap6mX-SePRqVz/view?usp=sharing

Unfortunately this file isn't accessible. In Google you'll need to change sharing to "Anyone with the link"

However, could you upload some screenshots of the text so we have a better idea. Are you playing with a language not in the original Theme Hospital?

**Comment by B-Mqn** [comment]
Sorry thought I had done it.  Should work now. 

I literally can't read the red writing popup in the bottom right or he fax writing. The machine uses  etc.  It's all terrible. 

Standard English setting
I tried different fonts and even changing my windows 10 scaling from 125% to 100 but still didn't change anything. 
The original files on dosbox show it fine but the game stutters too much.  This version works great I just can't read anything.

**Comment by lewri** [comment]
That's not the built in font. Did you receive any warning about corrupted game files in the command prompt window? Or set a font in the game's menu?

Could you also make sure the Font setting in the config.txt file is not set? (Should appear as just `unicode_font = nil -- [[X:\ThemeHospital\font.ttc]]`)
On Windows you can access this via Start > CorsixTH > CorsixTH AppData Folder > config.txt

Edit: As it's from CD I wonder if the files could have corrupted.

**Comment by B-Mqn** [comment]
I did have a corrupt font file.
I downloaded the demo and just copied the font files and its readable now!
but it still says the font file is corrupt.. im assuming the same one as its was unreadable what font number it was beforehand
This is after copying it from the demo download but it works.. 
https://drive.google.com/file/d/13Q98PTn9qgXeI9-vXLNEG9cMpHddX8N4/view?usp=sharing

I know the intro and lose files are corrupt as i remember it caused issues in dosbox or something and that was a fix back in the day and dont think they will affect the use. 

Thanks i should have read the error msg more carefully but with the font being bad i couldnt make out half of it

**Comment by lewri** [comment]
I think there's a conflict between the demo files and full game files now.

Demo has no INTRO.SM4, it should indeed be 32839KB in full
As above with LOSE, it doesn't exist in demo.
Demo has a different size FONT00 to full as I imagine the makeup of each file differs (and a different number of files).

As the full and demo now have mixed files that's leading to these warnings.

You may wish to check these files manually in the Theme Hospital folder after copying the contents from the CD
```
    ANIMS\AREA01V.SMK, 246KB
    ANIMS\WINGAME.SMK, 2019KB
    ANIMS\WINLEVEL.SMK, 328KB
    INTRO\INTRO.SM4, 32839KB
    QDATA\FONT00V.DAT, 1KB
    ANIMS\LOSE1.SMK, 987KB
```

If there is corruption it is unfortunately likely the CD has gone bad. There is no standard measure around for how long it takes for a CD to go bad as it can range from 1 year to decades depending on storage conditions (worth backing up any important CDs as ISOs somewhere before this happens.

I would suggest if the disc is just causing unended trouble I would have to look at purchasing it again. Unfortunately, this not the desired route to go down but on the plus side GOG have a sale on Theme Hospital currently for 76% off by the look of it (£1.19 for UK monies, not sure about others)
https://www.gog.com/en/game/theme_hospital
With the GOG version, you at least get a DRM free copy that you can download multiple times.

**Comment by TheCycoONE** [comment]
Can we move this to discussion.
